docs/wiki/deployment/file-integrity-monitoring.md

```diff
@@ -0,0 +1,82 @@
+As of osquery version 1.4.2, file integrity monitoring support was introduced
+for linux and darwin variants.  This module reads a list of directories to
+monitor from the osquery config and details changes and hashes to those
+selected files in the `file_changes` table.
+
+To get started with FIM (file integrity monitoring), you must first identify
+which files and directories you wish to monitor.
+Following the [wildcard rules](../development/wildcard-rules/), you can specify
+a directory or filename filter to limit the selection of files to monitor.
+
+For example, you may want to monitor `/etc` along with other files on a linux
+system.  After you identify your target files and directories you wish to monitor,
+add them to a new section in the config `file_paths`.  
+
+### Example Config
+
+```json
+{
+  "schedule": {
+    "crontab": {
+      "query": "select * from crontab;",
+      "interval": 300
+    },
+    "file_changes": {
+      "query": "select * from file_changes;",
+      "interval": 300
+    }
+  },
+  "file_paths": {
+    "homes": [
+      "/root/%%",
+      "/home/apprunner/%%"
+    ],
+    "all_etc": [
```

Comment:
Maybe change this to just `etc` to be consistent with `/tmp/%%` being just `tmp`?